src/test/java/com/wikia/webdriver/testcases/discussions/ZeroErrorStateTests.java

```diff
@@ -0,0 +1,230 @@
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void staffUserOnDesktopSeeProperMessageWhenUploadsEmptyPostsListPage() {
+    userSeeProperMessageWhenUploadsEmptyPostsListPage();
+  }
+
+  @Test(groups = "discussions-staffUserOnDesktopSeeProperMessageWhenUploadsNonExistingUserPostsPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void staffUserOnDesktopSeeProperMessageWhenUploadsNonExistingUserPostsPage() {
+    userOnDesktopSeeProperMessageWhenUploadsNonExistingUserPostsPage();
+  }
+
+  @Test(groups = "discussions-staffUserOnDesktopSeeProperMessageWhenUploadsEmptyPostDetailsPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void staffUserOnDesktopSeeProperMessageWhenUploadsEmptyPostDetailsPage() {
+    userOnDesktopSeeProperMessageWhenUploadsEmptyPostDetailsPage();
+  }
+
+  /**
+   * STAFF USERS ON MOBILE SECTION
+   */
+
+  @Test(groups = "discussions-staffUserOnMobileSeeProperMessageWhenUploadsEmptyReportedPostsPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5)
+  public void staffUserOnMobileSeeProperMessageWhenUploadsEmptyReportedPostsPage() {
+    userSeeProperMessageWhenUploadsEmptyReportedPostsPage();
+  }
+
+  @Test(groups = "discussions-staffUserOnMobileSeeProperMessageWhenUploadsEmptyPostsListPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5)
+  public void staffUserOnMobileSeeProperMessageWhenUploadsEmptyPostsListPage() {
+    userSeeProperMessageWhenUploadsEmptyPostsListPage();
+  }
+
+  @Test(groups = "discussions-staffUserOnMobileSeeProperMessageWhenUploadsNonExistingUserPostsPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5)
+  public void staffUserOnMobileSeeProperMessageWhenUploadsNonExistingUserPostsPage() {
+    userOnMobileSeeProperMessageWhenUploadsNonExistingUserPostsPage();
+  }
+
+  @Test(groups = "discussions-staffUserOnMobileSeeProperMessageWhenUploadsEmptyPostDetailsPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5)
+  public void staffUserOnMobileSeeProperMessageWhenUploadsEmptyPostDetailsPage() {
+    userOnMobileSeeProperMessageWhenUploadsEmptyPostDetailsPage();
+  }
+
+  /**
+   * TESTING METHODS SECTION
+   */
+
+  private void userOnDesktopSeeProperMessageWhenUploadsNonExistingUserPostsPage() {
+    ErrorMessages errorMessage = new UserPostsPage().open().getErrorMessages();
+    Assertion.assertTrue(errorMessage.isErrorMessagePresent());
+    Assertion.assertEquals(
+        errorMessage.getErrorMessageText(),
+        "Uh oh, looks like this page doesn't exist!\n"
+        + "Show Me All Discussions");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Show Me All Discussions" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)